libraries/Microsoft.Bot.Builder.AI.QnA/QnADialogResponseOptions.cs

```diff
@@ -41,5 +41,13 @@ public class QnADialogResponseOptions
         /// Get or set for Card no match response.
         /// </value>
         public Activity CardNoMatchResponse { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether DisplayPreciseAnswerOnly.
```

Comment:
>Gets or sets a value indicating whether DisplayPreciseAnswerOnly. [](http://example.com/codeflow?start=12&length=65)

Consider improving this summary: "Gets or sets a value that indicates whether the dialog response should display only precise answers".